api_app/analyzers_manager/observable_analyzers/greedybear.py

```diff
@@ -0,0 +1,40 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import requests
+
+from api_app.analyzers_manager.classes import ObservableAnalyzer
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class GreedyBear(ObservableAnalyzer):
+    def set_params(self, params):
+        self.__api_key = self._secrets["api_key_name"]
+        self.url = params.get("url", "https://greedybear.honeynet.org/api/enrichment")
```

Comment:
only the first part of the URL should be configurable that identifies the server (`https://greedybear.honeynet.org`).